folium/features.py

```diff
@@ -652,6 +682,69 @@ def __init__(self, html=None, icon_size=None, icon_anchor=None,
         self.className = class_name
 
 
+class Tooltip:
+    """
+    Creates a Tooltip object for adding to features to display text as a property a Map by executing a javascript
+    function when hovering the cursor over each feature.
+
+    Parameters
+    ----------
+        fields: list or tuple.
```

Comment:
The parameter names shouldn't be indentend, but have the same indentation level as the rest of the docstring